src/script/descriptor.cpp

```diff
@@ -533,25 +517,11 @@ std::unique_ptr<Descriptor> ParseScript(Span<const char>& sp, ParseScriptContext
         }
         return MakeUnique<MultisigDescriptor>(thres, std::move(providers));
     }
```

Comment:
This is in descriptor feature. I think we need to review this separately, if we need to keep this feature it should be extended to support coloured coins. otherwise we can remove it. I'll create a separate issue for this?

Reply:
I see. it is better to create another issue.